tests/customerSite/cart/updateCart/cartUpdatedAfterClickingMinusButton.spec.js

```
@@ -1,26 +1,38 @@
 import { test } from '../../../_fixtures/fixtures';
+import { allure } from 'allure-playwright';
+
 import { COFFEE_NAMES } from '../../../../src/constants';
 
-test('Cart updated correctly after clicking minus for drinks', async ({
-  cartPage,
-  menuPage,
-}) => {
-  await menuPage.open();
-  await menuPage.clickCoffeeCup(COFFEE_NAMES.cappuccino);
-  await menuPage.clickCoffeeCup(COFFEE_NAMES.espresso);
+test.describe('Cart > Update with removal', () => {
+  // Shared Allure labels for this file/suite
+  test.beforeEach(async () => {
+    await allure.parentSuite('Customer site');
+    await allure.suite('Cart');
+    await allure.subSuite('Cart update with removal');
+    await allure.epic('CoffeeCart Customer site');
+    await allure.feature('Cart');
+    await allure.severity('critical');
```

The `beforeEach` attaches parentSuite/suite/subSuite/epic/feature/severity via the module-level `allure`. This violates the same requirement quoted above: **"The labels must be attached in a way that is tied to the current test via the Playwright Allure adapter API so they appear in the report."**

Change the hook to use the Playwright testInfo-bound adapter. For example, accept `testInfo` in the hook signature and call `testInfo.allure.parentSuite(...)` etc., or use a fixture that exposes the per-test `allure`. Also ensure you fail fast if `testInfo.allure` is missing so reporter misconfiguration is visible.